index.html

```diff
@@ -1,17 +1,15 @@
-<!DOCTYPE html>
+<!doctype html>
```

Comment:
Prettier changed this to `doctype` and since the value is case-insensitive and there is no community consensus on UPPER vs lower, I left it.

\#DontTauntHappyFunBall